flake/dev/packages.nix

```diff
@@ -7,12 +7,46 @@
       # are derivations.
       checks = config.packages;
 
+      packages = config.testbeds;
+
       # Testbeds are virtual machines based on NixOS, therefore they are
       # only available for Linux systems.
-      packages = lib.mkIf pkgs.stdenv.hostPlatform.isLinux (
+      testbeds = lib.mkIf pkgs.stdenv.hostPlatform.isLinux (
         import ../../stylix/testbed {
           inherit pkgs inputs lib;
         }
       );
+
+      ci.buildbot = {
```

Comment:
The `ci.bulidbot` output does not cover the following valuable derivations from `stylix-check`:

- `apps.${system}.all-maintainers`
- `apps.${system}.doc`
- `checks.${system}.flake-parts-and-dev-flake-parts-version-consistency `
- `checks.${system}.nixpkgs-and-dev-nixpkgs-version-consistency `
- `checks.${system}.pre-commit`
- `checks.${system}.serve-docs`
- `checks.${system}.systems-and-dev-systems-version-consistency`
- `checks.${system}.treefmt`
- `devShells.${system}.default`
- `devShells.${system}.ghc`
- `formatter.${system}`